src/ethereum_test_tools/tests/test_fixtures/blockchain_london_invalid_filled.json

```diff
@@ -142,7 +142,62 @@
                 {
                     "rlp": "0xf902e1f901fea00e043cb2eb0339900f6199c0ab517e5be3a81d898fa58078ed8b866ddc60b010a01dcc4de8dec75d7aab85b567b6ccd41ad312451b948a7413f0a142fd40d4934794ba5e000000000000000000000000000000000000a069f3a735c7a7e1ea24a03a7107eba6a880d2d0251aaf24eaa7f109ece7969bf9a0ab28cd18f912c2177d3f787591ccc9ba7742c877cdeabe0098e7263ead8893c1a0976beb67b634171d419ef326220dfdda98074e3495940240a105e17643f0a4efb9010000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000830200000388016345785d8a0000830155442480a0000000000000000000000000000000000000000000000000000000000000000088000000000000000082029ff8ddb86c02f86901048203e88203e8830f424094cccccccccccccccccccccccccccccccccccccccc80820301c001a0720e2870881f8b0e285b7ec02c169f1165847bcb5f36ea5f33f3db6079854f63a04448266b715d7d99acd1e31dcab50d7119faa620d44c69b3f64f97d636634169b86d02f86a0105830186a08203e8830f424094cccccccccccccccccccccccccccccccccccccccd80820302c080a06c7fb2be7e001a210d72480522b9ebecade52d721360ce5242e34a6c05a02715a01220e3cb7418cd6294443b38d05f5ed9f2967b182d25c784e11e7863454b8f9bc0",
                     "expectException": "invalid transaction",
-                    "blocknumber": "3"
```

Comment:
Change here:
https://github.com/ethereum/execution-spec-tests/pull/322/commits/2458c859aea3e45e7efe8e016786e1d3ec30fe01#diff-42b8326ded3772a93dfdc20f92a48286faad3680c228f8908535e323359b3834L249

Reply:
the actual blocknumber is encoded in header, this is a debug field not required by the test format.

Reply:
Ahh nice catch, I figured it was redundant hence removed! 

Although we now have the case in the framework test:
`[valid_block_1, invalid_block_2, valid_block_3]`
where:
- valid_block_1 has `blocknumber=1`
- invalid_block_2 has no `blocknumber`
- valid_block_3 has `blocknumber=2`

I feel that this is a redundant test, i.e valid, invalid, valid, but it helped me catch the some bugs in my refactor here: #307 so it might be worth adding it back for consistency! I will check if it propagates in #307